packages/components/src/Autocomplete/Menu.tsx

```diff
@@ -52,6 +67,10 @@ export function Menu({
             </div>
             <div className={styles.text}>
               <Text>{option.label}</Text>
+              {option.details !== undefined && <span>{option.details}</span>}
```

Comment:
I like this. I think that also means that lists without headings can also use these. If so, please add a quick example that shows this option for a normal list.

Reply:
We might also want to think about the different layout cases and make sure we render a nice layout for each possible combination. If not, the other option is to only render `description` when we also have `details` or something like that.

Reply:
Yeah that sounds, good, adding this all in to the mdx.